src/routes/event.tsx

```diff
@@ -52,44 +54,65 @@ const noMatchesStyle = css`
 const Event = ({ eventKey }: Props) => {
   const matches = useEventMatches(eventKey)
   const eventInfo = useEventInfo(eventKey)
-  const newestIncompleteMatch = matches && nextIncompleteMatch(matches)
-
+  const newestIncompleteMatch = isData(matches) && nextIncompleteMatch(matches)
+  console.log(matches)
   return (
     <Page
       name={eventInfo?.name || <code>{eventKey}</code>}
       back="/"
       class={eventStyle}
     >
-      <div class={sectionStyle}>
-        <Heading level={2} class={headingStyle}>
-          Information
-        </Heading>
-        {eventInfo && <EventInfoCard event={eventInfo} />}
-        <Button href={`/events/${eventKey}/analysis`}>Analysis</Button>
-      </div>
+      {eventInfo instanceof NetworkError
+        ? 'Could not connect to server. Please check your connection.'
+        : eventInfo instanceof Error && (
+            <Fragment>
+              This event does not exist.
+              <Button href={`/`}>Existing Events</Button>
```

Comment:
Since there aren't any expressions embedded in the template literal, we can change this to regular quotes and remove the `{}`

```suggestion
              <Button href="/">Existing Events</Button>
```